### PR TITLE
vagrant: bump ubuntu-next VM image and increase jenkins timeouts

### PR DESCRIPTION
**Pull request by brb**
The new VM image includes the latest version of `cilium-{builder,runtime}` images: https://github.com/cilium/packer-ci-build/pull/208.

We will also increment the timeouts of all jenkins jobs so they don't fail due timeouts.

This should fix the timeouts we see in the net-next CI jobs.

The `ubuntu` image will be updated in a separate PR (it's still building), and it's not affected by the timeouts (we don't have that many tests in 4.9).

### CODE REVIEW

**Comment by brb** [comment]
test-me-please

**Review by pchaigno — APPROVED** [review]

**Review by borkmann — APPROVED** [review]

**Comment by aanm** [comment]
restart-ginkgo

File: jenkinsfiles/ginkgo.Jenkinsfile [hunk 1]
```diff
@@ -208,7 +208,7 @@ pipeline {
         }
         stage ("BDD-Test-PR"){
             options {
-                timeout(time: 130, unit: 'MINUTES')
+                timeout(time: 180, unit: 'MINUTES')
```
Comment (pchaigno) [inline jenkinsfiles/ginkgo.Jenkinsfile:211]: Doesn't the timeout at the top of this file also needs updating?

**Comment by aanm** [comment]
restart-ginkgo

**Review by pchaigno — APPROVED** [review]

**Review by tklauser — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Pull request merged**